### PR TITLE
Feature/event list

### PR DESCRIPTION
**Pull request by gaiisay**
User Story #1

### CODE REVIEW

File: components/EventCard/EventCard.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import EventCard from ".";
+
+describe("EventCard", () => {
+  it("displays all props", () => {
```
Comment (ahohnsen) [inline components/EventCard/EventCard.test.js:7]: I personally would like if you could be a bit more descriptive here :) e.g. renders name, date, time... of an event

File: components/EventCard/EventCard.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    const time = screen.getByTestId("time");
+    const name = screen.getByTestId("name");
+    const team = screen.getByTestId("team");
+
+    expect(date).toHaveTextContent(expectedDate);
+    expect(time).toHaveTextContent(expectedTime);
+    expect(name).toHaveTextContent(event.name);
+    expect(team).toHaveTextContent(teamName);
+
+    expect();
```
Comment (ahohnsen) [inline components/EventCard/EventCard.test.js:31]: I think you do not need this here 🤔 
```suggestion

```

File: components/EventCard/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const eventTime = event.startTime + " - " + event.endTime;
+
+  return (
+    <Card>
+      <StyledDate data-testid="date" dateTime={event.date}>
+        {eventDate}
+      </StyledDate>
+      <p data-testid="time">{eventTime}</p>
+      <h3 data-testid="name">{event.name}</h3>
+      <p data-testid="team">{teamName}</p>
```
Comment (ahohnsen) [inline components/EventCard/index.js:15]: See my comment above regarding test IDs.
Additionally, I think you could find more semantic elements than the `p` element as you are not really displaying paragraphs:

```suggestion
      <StyledDate dateTime={event.date}>
        {eventDate}
      </StyledDate>
      <time>{eventTime}</time>
      <h3>{event.name}</h3>
      <h4>{teamName}</h4>
```

File: components/EventCard/index.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+const Card = styled.article`
+  width: 90vw;
+  max-width: 500px;
+  padding: 1rem;
+  margin: 0.5rem;
+  display: grid;
+  grid-template-columns: 2rem 3fr;
+  grid-template-rows: repeat(3, 1fr);
```
Comment (ahohnsen) [inline components/EventCard/index.js:27]: You probably won't need this line 🙂 
```suggestion

```

File: components/GlobalStyles.js [hunk 5]
```diff
@@ -3,13 +3,21 @@ import { createGlobalStyle } from "styled-components";
...
+        align-items: center;
     }
 
     * {
         box-sizing: border-box;
+        padding: 0;
+        margin: 0;
+    }
+
+    h1, h2{
```
Comment (ahohnsen) [inline components/GlobalStyles.js:18]:
```suggestion
    h1, h2 {
```

File: pages/index.js [hunk 6]
```diff
@@ -1,3 +1,21 @@
+import useSWR from "swr";
+import EventCard from "../components/EventCard";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: events, error } = useSWR("/api/events", fetcher);
+  if (error) return <h1>There was an error</h1>;
+  if (!events) return <h1>...Loading...</h1>;
```
Comment (ahohnsen) [inline pages/index.js:8]: I suggest to add an empty line for better readability 🙂 
```suggestion
  const { data: events, error } = useSWR("/api/events", fetcher);
  
  if (error) return <h1>There was an error</h1>;
  if (!events) return <h1>...Loading...</h1>;
```

File: pages/index.js [hunk 7]
```diff
@@ -1,3 +1,21 @@
...
+
+  events.sort((a, b) => new Date(a.date) - new Date(b.date));
```
Comment (ahohnsen) [inline pages/index.js:10]: Please keep in mind, that you are mutating the original array here. This is not, what we want 🙈 
```suggestion
  const sortedEvents = events.sort((a, b) => new Date(a.date) - new Date(b.date));
```

File: pages/index.js [hunk 8]
```diff
@@ -1,3 +1,21 @@
...
+
+  return (
+    <>
+      <h1>⛹🏽‍♂️ Assist ⛹🏽‍♂️</h1>
+      <h2>Deine Veranstaltungen</h2>
+      {events.map((event) => (
```
Comment (ahohnsen) [inline pages/index.js:16]:
```suggestion
      {sortedEvents.map((event) => (
```

File: pages/index.js [hunk 9]
```diff
@@ -1,3 +1,21 @@
...
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: events, error } = useSWR("/api/events", fetcher);
+  if (error) return <h1>There was an error</h1>;
+  if (!events) return <h1>...Loading...</h1>;
+
+  events.sort((a, b) => new Date(a.date) - new Date(b.date));
+
+  return (
+    <>
```
Comment (ahohnsen) [inline pages/index.js:13]: Where is your `main` element? 🙃 
```suggestion
    <main>
```

File: pages/index.js [hunk 10]
```diff
@@ -1,3 +1,21 @@
...
+      <h1>⛹🏽‍♂️ Assist ⛹🏽‍♂️</h1>
+      <h2>Deine Veranstaltungen</h2>
+      {events.map((event) => (
+        <EventCard key={event.id} event={event} teamName="Your Team" />
+      ))}
+    </>
```
Comment (ahohnsen) [inline pages/index.js:19]: see above
```suggestion
    </main>
```

**Review by ahohnsen — CHANGES_REQUESTED** [review]
Hey Gai,

Well done 🙌🏼 I left a few comments for you. 🙂

File: pages/index.js (hunk 7 above)
Comment (gaiisay) [reply pages/index.js:10]: but wouldn't I then have to do something like `const sortedEvents = [...events].sort...` so the original array won't be mutated

Comment (ahohnsen) [reply pages/index.js:10]: 🤦🏼‍♀️ Sorry! You are right, of course. 🙂

**Review by ahohnsen — APPROVED** [review]
✨ Beautiful!

**Pull request merged**